pkg/resources/metrics.go

```diff
@@ -862,3 +864,24 @@ func reconcilePrometheusRule(ctx context.Context, client k8sclient.Client, ruleN
 
 	return rule, nil
 }
+
+func InstallationState(version string, toVersion string) string {
```

Comment:
@Boomatang , Seems following need to be changed,  “Installation”-> “Installing” and “Upgarde” -> “Upgrading”. buhttps://github.com/integr8ly/integreatly-operator/blob/b0066e0b36b8a05d34a91a4168d0f839c8049049/pkg/resources/metrics.go#L868-L873 . 
Probably it should be same as SyndesisPhaseInstalling , SyndesisPhaseUpgrading: /integreatly-operator/vendor/github.com/syndesisio/syndesis/install/operator/pkg/apis/syndesis/v1beta1/syndesis_types.go